src/libstd/time/duration.rs

```diff
@@ -168,15 +173,114 @@ impl Div<u32> for Duration {
 
 impl fmt::Display for Duration {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        match (self.secs, self.nanos) {
-            (s, 0) => write!(f, "{}s", s),
-            (0, n) if n % NANOS_PER_MILLI == 0 => write!(f, "{}ms",
-                                                         n / NANOS_PER_MILLI),
-            (0, n) if n % 1_000 == 0 => write!(f, "{}µs", n / 1_000),
-            (0, n) => write!(f, "{}ns", n),
-            (s, n) => write!(f, "{}.{}s", s,
-                             format!("{:09}", n).trim_right_matches('0'))
+        // FIXME: fmt::Formatter::with_padding() isn't public, and the pad() function applies
+        // precision in a way we don't want. At such time as it becomes reasonable to support
+        // padding without reimplementing fmt::Formatter::with_padding() here, this should be
+        // updated to support padding. In preparation for that, we're already writing to a
+        // stack buffer.
```

Comment:
I'm not sure what you mean. You're asking me to wrap it to 100, which is already the limit it's wrapped at. Did you mean to ask for a different line length?

FWIW, I typically wrap doc comments at 79 but leave code at 100. I didn't wrap this to 79 because it's not a doc comment.